lib/aff4.py

```diff
@@ -2393,6 +2405,10 @@ def Initialize(self):
       self.size = 0
       self.content_last = None
 
+    # This is the configurable default length for allowing Read to be called
+    # without a specific length
+    self.max_unbound_read = config_lib.CONFIG["Server.max_unbound_read_size"]
```

Comment:
in AFF4Stream class just add:

def Initialize(self):
   super(AFF4Stream, self).Initialize()
   self.max_unbound_read = config_lib.CONFIG["Server.max_unbound_read_size"]